### PR TITLE
Update Delegates.cs

### PR DESCRIPTION
**Pull request by zzfima**
Accordingly to Microsoft naming convention: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces

there shall be:
✔️ DO add the suffix "EventHandler" to names of delegates that are used in events. ✔️ DO add the suffix "Callback" to names of delegates other than those used as event handlers. ❌ DO NOT add the suffix "Delegate" to a delegate.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
Thanks for making these changes @zzfima We appreciate it.

I've reviewed the changes, and I'll :shipit: now.

EDIT:  I've approved this, but merging is blocked until you agree to the CLA. Once you do that, this PR will auto-merge.

**Comment by zzfima** [comment]
@dotnet-policy-service agree [company="{your company}"]

**Comment by zzfima** [comment]
@dotnet-policy-service agree

**Pull request merged**
